bench/bench.py

```diff
@@ -62,13 +61,13 @@ def __init__(self, path):
 		self.cwd = os.path.abspath(path)
 		self.exists = is_bench_directory(self.name)
 
+		self.apps_txt = os.path.join(self.name, "sites", "apps.txt")
+		self.excluded_apps_txt = os.path.join(self.name, "sites", "excluded_apps.txt")
```

Comment:
Moved from 3 line below to here because:
1. these two steps don't depend on any of them
2. `self.apps` creation now depends on `self.apps_txt`